src/mkdocs_caption/post_processor.py

```diff
@@ -36,9 +36,9 @@ def register_target(self, identifier: str, text: str, page: Page) -> None:
             "{page_title}",
             page.title,
         ).replace("{local_ref}", text)
-        self._regex_to_apply[rf'{page.file.src_path[:-3]}/#{identifier}"'] = (
+        self._regex_to_apply[rf'{page.file.src_path[:-3]}.html#{identifier}"'] = (
             re.compile(
-                rf'({page.file.src_path[:-3]}/#{identifier}".*?>)(</a>)',
+                rf'({page.file.src_path[:-3]}.html#{identifier}"[^>]*?>)(<\/a>)',
```

Comment:
```suggestion
                rf'({re.escape(page.file.src_path[:-3])}.html#{identifier}"[^>]*?>)(<\/a>)',
```

Another escape is needed here, afterwards the errors are gone from my point of view. (I checked the test setup and our original setup with the exchanged post_processor.py in my local dependencies.)

Reply:
Thanks a lot